config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```
@@ -158,6 +159,10 @@ presubmits:
           env:
             - name: FOCUS
               value: NodeConformance
+            - name: USE_DOCKERIZED_BUILD
+              value: true
```

maybe ?
```suggestion
              value: "true"
```

to address the lint issue ?

---

Done!